### PR TITLE
Enable templating of viewtopic caption

### PR DESCRIPTION
**Pull request by rica-carv**
Enable templating of viewtopic caption

### CODE REVIEW

File: e107_plugins/forum/forum_viewtopic.php [hunk 1]
```diff
@@ -615,7 +616,7 @@ function forumbuttons($thread)
 
 if ($forum->prefs->get('enclose'))
 {
-	$forumTitle = e107::pref('forum','title', LAN_PLUGIN_FORUM_NAME);
+	$forumTitle = varset($FORUMCAPTION) ? $tp->parseTemplate($FORUMCAPTION, TRUE, $sc) : e107::pref('forum','title', LAN_PLUGIN_FORUM_NAME);
```
Comment (CaMer0n) [inline e107_plugins/forum/forum_viewtopic.php]: Just use `!empty($FORUMCAPTION) `instead, and it should be good to go. Thanks.

Comment (rica-carv) [inline e107_plugins/forum/forum_viewtopic.php]: Donne.
Thought that it was better to use the core varset function....
Is the varset core function to be deprecated?

**Comment by CaMer0n** [comment]
Thank you! btw. For the future - I'm trying to use lowercase 'true' and 'false' and not "TRUE" and "FALSE"

**Comment by rica-carv** [comment]
Err, i copy that code from somewhere else, and it was already uppercase.
Anyway, i'll keep in mind to lowercase the 'TRUE' & 'FALSE' i find out when changing code...
Sorry...

**Pull request merged**